examples/host/simpleResolver/test_simpleResolver.py

```diff
@@ -95,7 +95,14 @@ def execute_cli(*args):
     all_args.extend(args)
     # We explicitly don't want an exception to be raised.
     # pylint: disable=subprocess-run-check
-    return subprocess.run(all_args, capture_output=True, encoding="utf-8", cwd=this_file.parent)
+    return subprocess.run(
+        all_args,
+        capture_output=True,
+        encoding="utf-8",
+        # Deliberately not same directory as the library JSON file, so
+        # relies on ${config_dir} interpolation.
+        cwd=this_file.parent.parent,
```

Comment:
[question] It's a long shot, but do we definitely want to make assumptions that we're still in a structure that has the `simpleResolver` dir?  Eg, if someone had copied the contents of `examples/host/simpleResolver` to `E:\`

Reply:
Or actually, do we even need to set this, as I think in CI, this is running from the repo root?

Reply:
Good idea, switched to tmpdir in bcd89b5f.  I think its worth explicitly setting, since we don't want to risk false negative failures, and it's good to be explicit that we're testing this.